Cargo.lock

```diff
@@ -995,10 +1001,10 @@ dependencies = [
  "clap 3.1.2",
  "clap_complete",
  "dirs 3.0.2",
- "fuel-asm",
+ "fuel-asm 0.2.0",
  "fuel-gql-client",
- "fuel-tx",
- "fuel-vm",
+ "fuel-tx 0.6.0",
+ "fuel-vm 0.5.0",
```

Comment:
When you see the `Cargo.lock` start to specify the version for dependencies like this, it's a hint that there might be two crates in the workspace that have different versions of the same dependency, or that we may have forgotten to update one or more crates' dependencies.

In this case, it looks like `sway-core`'s `fuel-asm` and `fuel-vm` deps might have been missed?

Reply:
I was wondering why sometimes there were numbers and sometimes not!

Reply:
Fixed in [01cafb3](https://github.com/FuelLabs/sway/pull/886/commits/01cafb3f7e5d6968e473dea025bcb909d23072c2)